tools/setup/emoji/emoji_names.py

```diff
@@ -1321,28 +1320,28 @@
     "2653": {"canonical_name": "pisces", "aliases": []},
     "1f194": {"canonical_name": "id", "aliases": []},
     "269b": {"canonical_name": "atom", "aliases": ["physics"]},
-    # japanese symbol
-    # '1f251': {'canonical_name': 'X', 'aliases': ['accept']},
     "2622": {"canonical_name": "radioactive", "aliases": ["nuclear"]},
     "2623": {"canonical_name": "biohazard", "aliases": []},
     "1f4f4": {"canonical_name": "phone_off", "aliases": []},
     "1f4f3": {"canonical_name": "vibration_mode", "aliases": []},
-    # '1f236': {'canonical_name': 'X', 'aliases': ['u6709']},
-    # '1f21a': {'canonical_name': 'X', 'aliases': ['u7121']},
-    # '1f238': {'canonical_name': 'X', 'aliases': ['u7533']},
-    # '1f23a': {'canonical_name': 'X', 'aliases': ['u55b6']},
-    # '1f237': {'canonical_name': 'X', 'aliases': ['u6708']},
+    # Japanese symbol. `canonical_name` taken from emojipedia.org.
+    "1f236": {"canonical_name": "japanese_not_free_of_charge_button", "aliases": ["u6709"]},
+    "1f250": {"canonical_name": "japanese_bargain_button", "aliases": ["ideograph_advantage"]},
+    "1f251": {"canonical_name": "japanese_acceptable_button", "aliases": ["accept"]},
+    "1f21a": {"canonical_name": "japanese_free_of_charge_button", "aliases": ["u7121"]},
```

Comment:
What's the story with the aliases that look like codes like `u7121`?

Reply:
Those are the Unicode names, e.g. “Squared CJK Unified Ideograph-7121” and are also the shortcodes used in Slack and Matrix.org.

Reply:
OK.  It's weird because very few other unicode symbols do this u7121 thing, but it doesn't seem worth diverging on.

Reply:
(This is some weird GitHub behavior: my comment appears above Tim's, and I'm pretty sure I wasn't seeing Tim's when I wrote it… but when I hover on the dates, it says Tim's comment was several hours before mine!

Anyway, Tim, that probably means you haven't seen my comment above.)